src/app/common/components/modals/dialog-create-or-update-wallet/dialog-create-or-update-wallet.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit, Inject } from '@angular/core';
+import {MatDialogRef, MAT_DIALOG_DATA} from '@angular/material/dialog';
+
+@Component({
+  selector: 'app-dialog-create-or-update-wallet',
+  templateUrl: './dialog-create-or-update-wallet.component.html',
+  styleUrls: ['./dialog-create-or-update-wallet.component.scss']
+})
+export class DialogCreateOrUpdateWalletComponent implements OnInit {
+
+    constructor(
+        public dialogRef: MatDialogRef<DialogCreateOrUpdateWalletComponent>,
+        @Inject(MAT_DIALOG_DATA) public data: { title: string, btn: string, inputData: string}
+    ) { }
+
+    ngOnInit(): void {
+    }
+
+    cancel(data: any): void {
```

Comment:
You should have two methods -> `save()` and `cancel()`. Also please do not use `any` - I will try to improve linter in the application to not allow this. You should basically never use it if really not necessary. 
Also please always add access modifier for methods - so this should be `public cancel(` not `cancel(`